src/web-api.js

```diff
@@ -151,38 +151,32 @@ class WebApi {
 
         _private.data[key] = bool;
       },
-      determineSecKey: function(timestamp) {
-        // calculate an outgoing signature
-        return new Signature(_private.data.partner_id, _private.data.api_key).generate_sec_key(timestamp || _private.data.timestamp);
-      },
       determineSignature: function(timestamp) {
         // calculate an outgoing signature
         return new Signature(_private.data.partner_id, _private.data.api_key).generate_signature(timestamp || _private.data.timestamp);
       },
       configurePrepUploadJson: function() {
-        var body =  {
+        const body = {
           file_name: 'selfie.zip',
           use_enrolled_image: _private.data.use_enrolled_image,
           timestamp: _private.data.timestamp,
           smile_client_id: _private.data.partner_id,
           partner_params: _private.data.partner_params,
           model_parameters: {},
-          callback_url: _private.data.callback_url
+          callback_url: _private.data.callback_url,
+          signature: _private.determineSignature().signature,
+          source_sdk: "javascript",
+          source_sdk_version: "2.0.0"
         };
-        if (options && options.signature) {
-          body.signature = _private.determineSignature().signature;
-        } else {
-          body.sec_key = _private.determineSecKey().sec_key;
-        }
         return JSON.stringify(body);
       },
       setupRequests: function() {
         // make the first call to the upload lambda
         var json = '';
```

Comment:
Oops looked like you missed this
```suggestion
        let json = '';
```